git-make-release

```diff
@@ -89,6 +131,54 @@ if [ "$NEXT_VERSION" == "$LAST_VERSION" ]; then
   exit 1
 fi
 
+# write new version to CMakeLists.txt, if --patch, --minor or --major was given
+if [ "${RELEASE_PATCH}" = "1" -o "${RELEASE_MINOR}" = "1" -o "${RELEASE_MAJOR}" = "1" ]; then
+  sed -i ${REPO_TOPLEVEL}/CMakeLists.txt \
+    -e "s|set(\${PROJECT_NAME}_MAJOR_VERSION .*|set(\${PROJECT_NAME}_MAJOR_VERSION ${MAJOR})|i"
+  sed -i ${REPO_TOPLEVEL}/CMakeLists.txt \
+    -e "s|set(\${PROJECT_NAME}_MINOR_VERSION .*|set(\${PROJECT_NAME}_MINOR_VERSION ${MINOR})|i"
+  sed -i ${REPO_TOPLEVEL}/CMakeLists.txt \
+    -e "s|set(\${PROJECT_NAME}_PATCH_VERSION .*|set(\${PROJECT_NAME}_PATCH_VERSION ${PATCH})|i"
+
+  # commit the version bump
+  git checkout -b ${USER}/version-bump-${NEXT_VERSION}
+  git add ${REPO_TOPLEVEL}/CMakeLists.txt
+  git commit -m "chore: Bump version to ${NEXT_VERSION}"
+  git push
+
+  # check if repo is on github or gitlab, then create a PR/MR using the glab/gh CLI
+  origin=`git remote get-url origin`
+  if [[ $origin = *gitlab* ]]; then
+    glab mr create --title "chore: Bump version to ${NEXT_VERSION}" --description "This MR bumps the version to ${NEXT_VERSION}" --target-branch master
+  elif [[ $origin = *github* ]]; then
+    gh pr create --title "chore: Bump version to ${NEXT_VERSION}" --body "This PR bumps the version to ${NEXT_VERSION}" --base master
+  else
+    echo "No gitlab or github remote found, please create a PR/MR manually."
+    exit
+  fi
+
+  # ask whether to merge the MR/PR and do so via the CLI
+  echo -n "Do you want to merge the PR/MR (y/N)? "
+  read ANSWER
+  if [ "$ANSWER" != "y" ]; then
+    echo "Aborted."
```

Comment:
Suggestion: `if  [ "${ANSWER,,}" != "y" ]; then` also matches the occasional "Y"

Reply:
done